 BinaryConverter/src/amicableN/AmicableNumberFinder.java

```diff
@@ -0,0 +1,46 @@
+package amicableN;
+import java.util.ArrayList;
+
+public class AmicableNumberFinder {
+	
+	private static int a = 220;
+	private static int b = 284;
+	
+	private static ArrayList<Integer> checked = new ArrayList<Integer>();
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		for(int i = 1; i < 10e6; i++) {
+			int j = allFactors(i);
+			if(allFactors(j) == i && !checked.contains(i) && !checked.contains(j) && i != j) {
+				checked.add(i);
+				checked.add(j);
+				
+				System.out.println(i + " " + j);
+			}
+		}
+		
+		
+		
+
+	}
+	public static int allFactors(int a) {
+		
+		ArrayList<Integer> list = new ArrayList<Integer>();
```

Comment:
I realized the reason why your code runs a lot slower than mine is that you need to create a list and then add them up. A list takes more time and memory to process than a `int` that just represents the value. You ran two `for loops` here while you only need one. It works, but see if you can make it faster.